packages/engine/Source/Scene/VoxelTraversal.js

```diff
@@ -224,7 +224,7 @@ function VoxelTraversal(
  */
 VoxelTraversal.prototype.findKeyframeNode = function (megatextureIndex) {
   return this._keyframeNodesInMegatexture.find(function (keyframeNode) {
-    return (keyframeNode.megatextureIndex = megatextureIndex);
```

Comment:
This seems like it could have caused some problems... Any idea if it fixed existing bugs?

Reply:
I introduced this in https://github.com/CesiumGS/cesium/pull/11754, and didn't notice the bug until I did more testing in this PR.